x/simulation/operation.go

```diff
@@ -22,36 +23,38 @@ type OperationEntry struct {
 	Height    int64           `json:"height" yaml:"height"`
 	Order     int64           `json:"order" yaml:"order"`
 	Operation json.RawMessage `json:"operation" yaml:"operation"`
+	BlockTime int64           `json:"block_time" yaml:"block_time"`
 }
 
 // NewOperationEntry creates a new OperationEntry instance
-func NewOperationEntry(entry string, height, order int64, op json.RawMessage) OperationEntry {
+func NewOperationEntry(entry string, blockTime time.Time, height, order int64, op json.RawMessage) OperationEntry {
```

Comment:
This will be API breaking when users upgrade right?

Reply:
Although this function is public, it is used internal to the legacy **sims framework only**.  Any changes here should not affect any module code for v0.53. 
You can argue that any change to a public method is breaking the code compatibility but as long as we are not planning to run v0.50 modules with v0.53 SDK, this has no practical effect. 
The integration with the legacy sims framework in the modules is on a higher level with `simtypes.WeightedOperations` and `simtypes.Operation`. 

I am going to more code compatibility breaking changes in a new PR. But they would only affect sims and not consensus or rpc.